pontoon/base/templates/translate.html

```diff
@@ -125,9 +125,13 @@
             {% endif %}
 
             {% if user.is_authenticated() %}
+                {% if user.logged_via('fxa') and not user.logged_via('persona') %}
```

Comment:
This basically makes sure that if the user is logged in with Firefox Account, it hasn't connected it with Persona yet, right?